src/ParseQuery.js

```diff
@@ -993,7 +993,7 @@ class ParseQuery {
   *
   * In order to sort you must use select and ascending ($score is required)
   *  <pre>
-  *   query.fullText('term');
+  *   query.fullText('field', 'term', { $language: "spanish" });
```

Comment:
For documentation purposes leave an example of with and w/o options.

Docs are auto generated by using `npm run docs`. Use it to check your formatting.

`$language` is now `language`